generator/subaru/subaru_forester_2017.dbc

```diff
@@ -0,0 +1,15 @@
+CM_ "IMPORT _subaru_preglobal_2015.dbc"
+
+BO_ 355 ES_DashStatus: 8 XXX
+ SG_ Not_Ready_Startup : 4|2@1+ (1,0) [0|3] "" XXX
+ SG_ Cruise_On : 16|1@1+ (1,0) [0|1] "" XXX
+ SG_ Cruise_Set_Speed : 24|8@1+ (1,0) [0|255] "" XXX
+ SG_ Counter : 40|3@1+ (1,0) [0|7] "" XXX
+ SG_ Cruise_Activated : 54|1@1+ (1,0) [0|1] "" XXX
+
+BO_ 881 Steering_Torque: 8 XXX
+ SG_ Steering_Motor_Flat : 0|10@1+ (32,0) [0|1000] "" XXX
+ SG_ Steer_Torque_Output : 16|11@1- (-32,0) [-1000|1000] "" XXX
+ SG_ LKA_Lockout : 27|1@1+ (1,0) [0|1] "" XXX
+ SG_ Steer_Torque_Sensor : 29|11@1- (-1,0) [-1000|1000] "" XXX
```

Comment:
Hmm, that's odd that they'd just use the same message except flip the sign on this signal. Are you sure about this one?

Reply:
Before making the change, I confirmed from cabana drive that the user torque signal is reversed for Foresters. I have two Forester users who confirmed assisted lane change works correctly now. They had to nudge the wheel to the opposite side to initiate lane change before